src/Microsoft.IdentityModel.Tokens/Validators.cs

```diff
@@ -38,6 +38,36 @@ namespace Microsoft.IdentityModel.Tokens
     /// </summary>
     public static class Validators
     {
+        /// <summary>
+        /// Validates if a given algorithm for a <see cref="SecurityKey"/> is valid.
+        /// </summary>
+        /// <param name="algorithm">The algorithm being considered if it is valid</param>
```

Comment:
Maybe rephrase this comment as "the algorithm to be validated".

Reply:
that is more clear, good idea.